README.rst

```diff
@@ -104,7 +104,8 @@ Most of the API is available:
     client.broadcasts.get_round
     client.broadcasts.update_round
     client.broadcasts.get_round_pgns
-    client.broadcasts.get_pgns
```

Comment:
this function still exists so why remove it?

Reply:
Oh, my bad.